core/src/cluster_info_vote_listener.rs

```diff
@@ -426,6 +431,15 @@ impl ClusterInfoVoteListener {
                         &verified_vote_packets,
                     )?;
                 }
+                // Check if we've crossed the feature boundary
+                if !is_tower_full_vote_enabled {
+                    is_tower_full_vote_enabled = bank_forks
+                        .read()
+                        .unwrap()
+                        .root_bank()
+                        .feature_set
+                        .is_active(&allow_votes_to_directly_update_vote_state::id());
```

Comment:
I think we should just check on a fixed interval like the `BANK_SEND_VOTES_LOOP_SLEEP_MS`. Maybe like every few seconds or so, so we don't have to rely on the `receive_and_process_vote_packets()` returning nothing

Reply:
This check is already inside `if time_since_lock.elapsed().as_millis() > BANK_SEND_VOTES_LOOP_SLEEP_MS {` so I think we're good there

Reply:
ah shit, misread because of the github code collapsing